docs/monitoring/flow.rst

```diff
@@ -0,0 +1,68 @@
+
+Both the raw statistics and the statistics history can be used as data feeds into the internal Fledge notification service. This means that we can use the Fledge mechanisms for capturing events and sending notifications to many destinations to monitor the flow rates of our south and north services.
+
+Monitor Low North Flow Rate
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As an example we will setup a notification on the flow rate north dropping below a low water mark. For this we will use the threshold notification rule and use the statistics history as the source of data. Let's assume we want to be notified when we send less than 400 readings north in a any given minute. Given our default 15 second statistics collection schedule we can either change that to be once a minute or we can set our threshold lower to 100 readings in every 15 seconds. For this example we will do the later.
+
++--------------------+
+| |MonitorNorthRate| |
++--------------------+
+
+When can then choose one of Fledge's many supported delivery plugins to have this notification sent to the destination that we require. For the purpose of this example we will use the ZenDesk plugin to raise a problem ticket to have the cause of the issue investigated.
+
++------------------+
+| |MonitorZendesk| |
++------------------+
+
+We can then decide how we want this to trigger, it is probably most appropriate to have the alert trigger first when we drop below that threshold and then be cleared when we rise above that threshold. We also do not wish to have it re-trigger again too soon, so we will set the re-trigger interval to 3600 seconds, or every hour. This way we do not raise excessive number of tickets.
+
++------------------+
+| |MonitorTrigger| |
++------------------+
+
+Detecting Fluctuating Flows
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Another useful notification rule plugin that can be used when processing statistical data is the *Simple-Sigma* plugin. This plugin uses statistical methods to collect the mean and standard deviations of a data stream and then detect when it sees values that are a long way outside of the standard deviation from the mean, usually three times the standard deviation.
+
++---------------------+
+| |MonitoredBuffered| |
++---------------------+
+
+In the above example it is shown monitoring the BUFFERED statistic, the is the number of readings written to the storage layer and represents the aggregate of all the data read by all of the south services. This can then be used to monitored if the ingest for the Fledge in total moves outside of the observed operating criteria that has been measured previously. 
```

Comment:
- Start new sentence at "This is the number of readings written to the storage layer..." Note that the first word should be "This" and not "The."
- "This can then be used to monitored if the ingest..." -> "This can then be used to monitor if the ingest..."